packages/studio/src/store/slices/accountContent/createAccountContentSlice.ts

```
@@ -19,6 +24,25 @@ const createAccountContentSlice: SliceCreator<AccountContentSlice> = (set) => ({
       console.error("Unable to fetch account content.", e);
     }
   },
+  async fetchEntities(entityType: string, pageNum: number): Promise<void> {
```

can we add tests for this?

---

It's not really doing anything except passing through params and we already have tests for fetchInitialEntities, but I can add a test if you'd like

---

I think it would be helpful to make sure the `entitiesRecord` is being updated correctly for new batches of entities

---

Yup there's a test now!